### PR TITLE
gcp - fix sql instance resource

### PR DESCRIPTION
**Pull request by PMitrafanau**
- Modified keys used for retrieval of data from an event in the `SqlInstance.resource_type.get`. Fixes `KeyError` exceptions when auditing sql instances. E.g.:
```yaml
policies:
  - name: serverless-sql-instance-stop-v2
    resource: gcp.sql-instance
    mode:
      type: gcp-audit
      methods:
        - "cloudsql.instances.create"
    actions:
      - stop
```
- Removed excess record flight which was used by tox because project wasn't explicitly specified in the test

### CODE REVIEW

**Comment by kapilt** [comment]
there's some flake8 issues in the tests dir, else lgtm

**Comment by kapilt** [comment]
gonna try opening and closing this a couple of times per @stefangordon that should eventually retrigger azure pipelines :-(

File: tools/c7n_gcp/tests/test_sql.py [hunk 1]
```diff
@@ -36,8 +37,8 @@ def test_sqlinstance_get(self):
              'resource': 'gcp.sql-instance'},
             session_factory=factory)
         instance = p.resource_manager.get_resource(
-            {"project": "cloud-custodian",
-             "name": "brenttest-2"})
+            {'project_id': 'cloud-custodian',
+             'database_id': 'cloud-custodian:brenttest-2'})
```
Comment (kapilt) [inline tools/c7n_gcp/tests/test_sql.py]: could clarify this is actually in the event for api calls?

Comment (PMitrafanau) [reply tools/c7n_gcp/tests/test_sql.py]: Yes. It is. Here is excerpt from a log entry:
```json
{
  "protoPayload": {
    ...
     "methodName": "cloudsql.instances.create",
     "resourceName": "instances/mitrop-custodian:test-mysql",
    ...
  },
   "resource": {
     "type": "cloudsql_database",
     "labels": {
       "project_id": "mitrop-custodian",
       "database_id": "mitrop-custodian:test-mysql",
       "region": "us-central"
     }
   },
   ...
}
```

Comment (kapilt) [reply tools/c7n_gcp/tests/test_sql.py]: thanks for confirming.

**Pull request merged**